ananas/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,44 @@
+    <string name="iamutkarshtiwari_github_io_ananas_cancel">Cancelar</string>
+    <string name="iamutkarshtiwari_github_io_ananas_confirm">Confirmar</string>
+    <string name="iamutkarshtiwari_github_io_ananas_crop">Recortar</string>
+    <string name="iamutkarshtiwari_github_io_ananas_done">HECHO</string>
+    <string name="iamutkarshtiwari_github_io_ananas_edit">Editar</string>
+    <string name="iamutkarshtiwari_github_io_ananas_eraser">Borrador</string>
+    <string name="iamutkarshtiwari_github_io_ananas_exit_without_save">La imagen no esta guardada. Quiere salir?</string>
+    <string name="iamutkarshtiwari_github_io_ananas_filter">Filtros</string>
+    <string name="iamutkarshtiwari_github_io_ananas_fit_image">ORIGINAL</string>
+    <string name="iamutkarshtiwari_github_io_ananas_free_size">LIBRE</string>
+    <string name="iamutkarshtiwari_github_io_ananas_images">%d imagenes</string>
+    <string name="iamutkarshtiwari_github_io_ananas_input_hint">Campo de Texto ...</string>
+    <string name="iamutkarshtiwari_github_io_ananas_library_name">Editor de Foto</string>
+    <string name="iamutkarshtiwari_github_io_ananas_load_error">Fallo la carga de imagen!</string>
+    <string name="iamutkarshtiwari_github_io_ananas_loading">Cargando...</string>
+    <string name="iamutkarshtiwari_github_io_ananas_more">Mas</string>
+    <string name="iamutkarshtiwari_github_io_ananas_no_images">Imagen no encontrada</string>
+    <string name="iamutkarshtiwari_github_io_ananas_not_selected">Porfavor elija una imagen para editar</string>
+    <string name="iamutkarshtiwari_github_io_ananas_opacity">Opacidad</string>
+    <string name="iamutkarshtiwari_github_io_ananas_pick_a_color">Seleccione un color</string>
+    <string name="iamutkarshtiwari_github_io_ananas_ratio16_9">16:9</string>
+    <string name="iamutkarshtiwari_github_io_ananas_ratio3_4">3:4</string>
+    <string name="iamutkarshtiwari_github_io_ananas_ratio4_3">4:3</string>
+    <string name="iamutkarshtiwari_github_io_ananas_ratio9_16">9:16</string>
+    <string name="iamutkarshtiwari_github_io_ananas_rotate">Rotar</string>
+    <string name="iamutkarshtiwari_github_io_ananas_saturation">Saturacion</string>
+    <string name="iamutkarshtiwari_github_io_ananas_save_error">Error en guardado!</string>
+    <string name="iamutkarshtiwari_github_io_ananas_saving_image">Guardando...</string>
+    <string name="iamutkarshtiwari_github_io_ananas_size">Tamaño</string>
+    <string name="iamutkarshtiwari_github_io_ananas_smooth">Suavidad</string>
+    <string name="iamutkarshtiwari_github_io_ananas_stickers">Stickers</string>
+    <string name="iamutkarshtiwari_github_io_ananas_text">Texto</string>
+    <string name="iamutkarshtiwari_github_io_ananas_white_skin">Tono de piel</string>
+    <string name="materialcolorpicker__btnSelectColor">Seleccionar</string>
+    <string name="materialcolorpicker__errHex">Formato de Codigo Hex incorrecto</string>
```

Comment:
Please add tests for these as well: 
https://github.com/iamutkarshtiwari/Ananas/blob/master/ananas/src/main/res/values/strings.xml#L98-L108
https://github.com/iamutkarshtiwari/Ananas/blob/master/ananas/src/main/res/values/strings.xml#L41-L54
https://github.com/iamutkarshtiwari/Ananas/blob/master/ananas/src/main/res/values/strings.xml#L80-L87

Reply:
Done.